docs/index.html

```
@@ -0,0 +1,524 @@
+
+<!-- ============ EXAMPLE ============ -->
+<section id="example">
+  <div class="wrap center">
+    <p class="eyebrow">What a test looks like</p>
+    <h2>Real tests, not pseudocode</h2>
+    <p class="lede">Straight out of the official sample project — nothing trimmed to look good here.</p>
+  </div>
+  <div class="wrap" style="max-width:860px;margin-top:42px">
+    <div class="code">
+      <div class="tabs" role="tablist" aria-label="Example tests">
+        <button class="tab" role="tab" id="t-api" aria-controls="p-api" aria-selected="true">API</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)